pkg/minishift/addon/parser/addon_parser.go

```diff
@@ -180,12 +180,25 @@ func (parser *AddOnParser) parseCommands(scanner *bufio.Scanner) ([]command.Comm
 }
 
 func createMetaData(header []string) (addon.AddOnMeta, error) {
-	regex, _ := regexp.Compile(`^# ?(.*):(.*)`)
-	metaMap := make(map[string]string)
+	regex, _ := regexp.Compile(`^# ?([a-zA-Z-]*):(.*)`)
+	metaMap := make(map[string]interface{})
+	var key string
+	var value []string
 	for _, line := range header {
 		matches := regex.FindAllStringSubmatch(line, -1)
+		if len(matches) == 0 && key == "Description" {
```

Comment:
that's a bit fragile. You are using twice the hard coded string "Description". At the very least it should be a constant in this file, or why not adding two constants in _addon_meta.go_ (DescriptionMetaTagName and NameMetaTagName). You can then reference this constant here and you can add these constants to _requiredMetaTags_ in _addon_meta.go_.

The more generic solution is to introduce as mentioned a MetaTagType, but maybe let's not go there.